honeybee_energy/properties.py

```diff
@@ -0,0 +1,46 @@
+"""Energy Properties."""
+from .boundarycondition import BoundaryConditions
+
+
+class EnergyProperties(object):
+    """Energy Properties for Honeybee geometry."""
+
+    def __init__(self, srf_type):
+        self.construction = srf_type.energy_construction
+        self.boundary_condition = BoundaryConditions().outdoors
```

Comment:
Do wee need to specify a separate construction and boundary condition instance for every single face?  This seems like it will greatly increase the memory needed when you have thousands of surfaces with the same construction and boundary condition. Is there some way that we could just create one instance for each of the default boundary conditions and constructions and have the various faces reference that instance?

Reply:
That's a good suggestion. As discussed this should be possible but we need a strategy for how to implement it with the current structure.

Reply:
Does this mean that you don't plan to address it in this PR and we should just merge this one in?  I could take a stab at trying to implement it if we merge this one in.

Reply:
Let me give it a try first today and you can modify it as needed.

Reply:
I changed the boundary condition to return the same instance when possible.
Construction is also set to construction-set structure similar to what you have suggested and documented in UrbanOpt document. All the code is MVP to show the workflow and is missing all the types and boundary conditions.

Reply:
This looks good @mostaphaRoudsari . Thank you.